vendor/licenses/grammar/ballerina-grammar.txt

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Have you been using the script as per [the contribution guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#changing-the-source-of-a-syntax-highlighting-grammar) to add this file? It looks weird, but it might be an issue with the script.

Reply:
Yes, I used the script to update the grammar.